lib/ansible/galaxy/collection/concrete_artifact_manager.py

```diff
@@ -280,21 +338,23 @@ def get_direct_collection_meta(self, collection):
         self._artifact_meta_cache[collection.src] = collection_meta
         return collection_meta
 
-    def save_collection_source(self, collection, url, sha256_hash, token):
-        # type: (Candidate, str, str, GalaxyToken) -> None
+    def save_collection_source(self, collection, url, sha256_hash, token, signatures_url, signatures):
+        # type: (Candidate, str, str, GalaxyToken, str, List[Dict[str, str]]) -> None
```

Comment:
It shouldn't be, it's the same URL we use to find the download_url. Fixed the comment and updated `get_collection_version_metadata` in GalaxyAPI to do `data['href']` instead of `data.get('href')`.